.github/workflows/build.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           # Since no browser is installed during this CI run, plotly fails to
           # produce HTML output, see https://github.com/plotly/plotly.py/blob/2c2dd6ab2eeff73c782457f33c590c1d09a97625/packages/python/plotly/plotly/io/_renderers.py#L532
           export PLOTLY_RENDERER=browser
-          mkdocs build
+          mkdocs build --strict
```

Comment:
If any markdown pages exists but is not referenced in the navigation, the build will fail.